src/views/validated-designs/__tests__/server.ts

```diff
@@ -0,0 +1,81 @@
+import { getHvdCategoryGroupsPaths } from '../server'
+import { HvdCategoryGroup } from '../types'
+
+// mock the extract-hvd-content file because importing from it cause it's self-invoking function to perform a side effect
+jest.mock('../../../../scripts/extract-hvd-content', () => ({
+	__esModule: true, // this property makes it work
+	HVD_CONTENT_DIR: '',
+}))
+
+const mockedHvdCategoryGroups: HvdCategoryGroup[] = [
```

Comment:
WDYT about making a __fixtures__ folder with this object so the file is shorter?